Source/CesiumRuntime/Private/CesiumGltfComponent.cpp

```diff
@@ -286,6 +287,21 @@ static void computeTangentSpace(TArray<FStaticMeshBuildVertex>& vertices) {
   genTangSpaceDefault(&MikkTContext);
 }
 
+static void computeFakeNormals(
+    const TArray<uint32_t>& indices,
+    TArray<FStaticMeshBuildVertex>& vertices) {
+  // Compute flat normals
+  for (int i = 0; i < indices.Num(); i += 3) {
+    FStaticMeshBuildVertex& v0 = vertices[i];
+    FStaticMeshBuildVertex& v1 = vertices[i + 1];
+    FStaticMeshBuildVertex& v2 = vertices[i + 2];
+
+    v0.TangentX = v1.TangentX = v2.TangentX = TMeshVector3(0.0f);
+    v0.TangentY = v1.TangentY = v2.TangentY = TMeshVector3(0.0f);
+    v0.TangentZ = v1.TangentZ = v2.TangentZ = TMeshVector3(0.0f, 0.0f, 1.0f);
```

Comment:
I think maybe using +Z as the normal isn't quite right. It's fine when the tileset is located near the georeference origin. But what if the origin is on one side of the globe, and the tileset is on the other? Then the tileset would only be lit when the sun is below the horizon, which is likely to surprise users.

So I think instead of +Z, the normal at each point should be set to the ellipsoid surface normal at that point, transformed to the mesh's coordinate system.

Basically you need to get each vertex position, transform it to ECEF using the `transform` passed into this method, and then call `Ellipsoid::WGS84.geodeticSurfaceNormal` to get the surface normal at that point. Then that surface normal has to be transformed from ECEF back to Unreal coordinates using the inverse of that passed-in `transform` (don't forget to construct the vector to transform as `glm::dvec4(ecefNormal, 0.0)` before multiplying it with the inverse of the transform, because it's a direction not a position).

Reply:
OK, I believe I implemented the above in code. I'm not sure how to visualize the normals. However the code is straightforward enough. If I invert the normals the buildings get dark. Also, I disabled the dynamic shadows:
![result](https://user-images.githubusercontent.com/97980867/213025259-d840aa35-742b-4d52-b03b-cbfa4f08f2df.gif)